src/components/chat/Message.js

```diff
@@ -7,6 +9,9 @@ class Message extends Component {
     let { message, username } = this.props;
     let fromMe = message.from === username;
     let messageClass = fromMe ? 'chat-outgoing' : 'chat-incoming';
+    
+    let RFCDate = message.key.split('-')[0];
```

Comment:
@iamrobinpham FYI: soon, a timestamp will come from the server and we won't need to make assumptions about the `key`.

Reply:
Sweet, I'll modify the PR once that happens

Reply:
Actually, you should use `const` instead of `let` if you know that the variable will not be reassigned or redeclared which is true for basically all of the declarations in the render function.